lib/unleash_proxy_client_flutter.dart

```diff
@@ -456,6 +459,8 @@ class UnleashClient extends EventEmitter {
 
     metrics.start();
 
+    lastRefreshTimestamp = await _getLastRefreshTimestamp();
```

Comment:
This is the main fix. We should read the TTL from local storage in a start method just before the initial fetch toggles. Previously we were reading the last refresh timestamp too early when the context wasn't fully populated. It resulted in current context having only sessionId and missing e.g. userId. At the same time the local storage context had all the fields.